File: Advertisements.Web/UI/src/app/models/resource.ts
export class Resource{
    Id : number;
    Url : string;
    AdvertisementId :number;
    constructor (id:number, url:string, advertisementId:number){
      this.Id = id;
      this.Url = url;
      this.AdvertisementId = advertisementId;
    }
  }

File: Advertisements.DataAccess/Entities/Resource.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Advertisements.DataAccess.Entities
{
    public class Resource : IEntity
    {
        public bool IsDeleted { get; set; }

        public int Id { get; set; }

        public string Url { get; set; }

        [ForeignKey("Advertisement")]
        public int AdvertisementId { get; set; }

        public virtual Advertisement Advertisement { get; set; }
    }
}